recipes/modestimage/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "ModestImage" %}
+{% set version = "0.2" %}
+{% set sha256 = "4b71af469f0c4ba73a915c6a96040cc54e81c9ce0c793e6d3e6bb72a99cb53dc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib
```

Looks like this doesn't need Qt:
```suggestion
    - matplotlib-base
```

---

I don't think we need Qt for that package, good idea.